packages/web/src/components/GoogleAuth/styles.scss

```diff
@@ -0,0 +1,7 @@
+.googleLoginBtn {
```

Comment:
camelCase classes is commonly use for ids. I'd suggest rename it to google-login-btn

Reply:
Guys, are you going to fix the other minors in one of the google-related PRs?

Vertical alignment of the name in the header:
![image](https://user-images.githubusercontent.com/16776066/87961170-05e96400-cabe-11ea-8812-824f87a7c0b3.png)

Menu items pointing nowhere from non-main pages:
![image](https://user-images.githubusercontent.com/16776066/87961230-18fc3400-cabe-11ea-9dca-eb87c54b80b4.png)

Horizontal scroll on the main:
![image](https://user-images.githubusercontent.com/16776066/87961385-4e088680-cabe-11ea-84ab-d654fb2d548c.png)

Reply:
OK, I am fixing this one in my profile-related story:
![image](https://user-images.githubusercontent.com/16776066/87964501-03d5d400-cac3-11ea-9318-548dc8274ad9.png)

Reply:
Fixing them here:
https://github.com/dubbha/croton/pull/68
Please take a look to avoid conflicts

Reply:
It is not related to social login. 
Thanks for fixes.